pandas/src/algos_groupby_helper.pxi.in

```diff
@@ -669,6 +669,72 @@ def group_cumsum(numeric[:, :] out,
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
+def group_cummin(numeric[:, :] out,
+                 numeric[:, :] values,
+                 int64_t[:] labels,
+                 numeric[:, :] accum):
+    """
+    Only transforms on axis=0
+    """
+    cdef:
+        Py_ssize_t i, j, N, K, size
+        numeric val, min_val
+        int64_t lab
+
+    N, K = (<object> values).shape
+    accum = np.empty_like(accum)
+    accum[:] = _int64_max
```

Comment:
Yeah, probably the easiest way would be to add to the templated code - look at `group_min`/`max` - it is already dumping the max/min value into the template.

The other option (just fyi) - `numeric` here is a cython [fused type](http://cython.readthedocs.io/en/latest/src/userguide/fusedtypes.html#type-checking-specializations) - you can check and specialize on it like this.
```
if numeric is int64_t then:
    ...
elif numeric is float32_t then:
    ...
```